src/views/RoofSelectionView.vue

```diff
@@ -1,19 +1,82 @@
 <script setup lang="ts">
 import { useViewsStore } from '@/stores/views'
-import { onMounted } from 'vue'
+import { computed, inject, onMounted, ref } from 'vue'
 import { viewList } from '@/model/views.model'
 import { usePanelsStore } from '@/stores/panels'
+import FigureEnergy from '@/components/roof_selection/FigureEnergy.vue'
+import AddressTooltip from '@/components/roof_selection/AddressTooltip.vue'
+import TextDidYouKnow from '@/components/roof_selection/TextDidYouKnow.vue'
+import type { RennesApp } from '@/services/RennesApp'
+import RoofSelectionTooltip from '@/components/roof_selection/RoofSelectionTooltip.vue'
+import { useAddressStore } from '@/stores/address'
+import {
+  cleanHighlightedRoofs,
+  disableSelectRoofInteraction,
+  enableSelectRoofInteraction,
+} from '@/services/interactionUtils'
 
-const viewStore = useViewsStore()
+const rennesApp = inject('rennesApp') as RennesApp
 const panelsStore = usePanelsStore()
+const viewStore = useViewsStore()
+const addressStore = useAddressStore()
+let isOpen = ref(true)
+let isBuildingSelectionActive = ref(false)
+let addressClosedByUser = ref(false)
+
+let DISTANCE_MAX_FOR_SELECTION = 600
 
 onMounted(() => {
   viewStore.setCurrentView(viewList['roof-selection'])
-  panelsStore.setTypePanelDisplay('left')
+  panelsStore.setTypePanelDisplay('float-left')
   panelsStore.isCompletelyHidden = true
+  rennesApp
+    .get3DMap()
+    .getScene()
+    .postRender.addEventListener(() => {
+      let cameraDistance = rennesApp.getCurrentDistance()!
+      if (
+        isBuildingSelectionActive.value &&
+        cameraDistance > DISTANCE_MAX_FOR_SELECTION
+      ) {
+        isBuildingSelectionActive.value = false
```

Comment:
When we click on building on the view roof-selection, the view change for roof-selected-information. 
But this postRender listener is steal here. And the value of isBuildingSelectionActive.value is always empty. 
So the 'if' part is never call on roof-selected-information
Maybe don't use  isBuildingSelectionActive of RoofSelectionView but store it in a store

Reply:
It's not unselected, its remove the interaciton